centreon/www/include/configuration/configServers/DB-Func.php

```
@@ -381,6 +381,8 @@ function duplicateServer(array $server, array $nbrDup): void
         $rowServer['ns_activate'] = '0';
         $rowServer['is_default'] = '0';
         $rowServer['localhost'] = '0';
+        $rowServer['vmware_updated'] = '0';
+        $result->closeCursor();
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$result'. (undefined)

(UnusedLocalVariable)